test/skills/core.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from re import error
+from os.path import join, dirname
+
+from mycroft.skills.core import load_regex_from_file, load_regex
+from mycroft.util.log import getLogger
+
+__author__ = 'eward'
+logger = getLogger(__name__)
+
+
+class MockEmitter(object):
+    def __init__(self):
+        self.reset()
+
+    def emit(self, message):
+        self.types.append(message.message_type)
+        self.results.append(message.metadata['regex'])
+
+    def get_types(self):
+        return self.types
+
+    def get_results(self):
+        return self.results
+
+    def reset(self):
+        self.types = []
+        self.results = []
+
+
+class MycroftSkillTest(unittest.TestCase):
+    emitter = MockEmitter()
+    path = join(dirname(__file__), 'regex_test')
+
+    def check_load_from_file(self, path, regex_list=[]):
+        load_regex_from_file(join(self.path, path), self.emitter)
+        self.check_emitter(self.emitter, regex_list)
+
+    def check_load(self, path, regex_list=[]):
+        load_regex(path, self.emitter)
+        self.check_emitter(self.emitter, regex_list)
+
+    def check_emitter(self, emitter, regex_list):
+        for type in emitter.get_types():
```

Comment:
Since `type` is a native built-in python object, it's a good idea to rename this var to something.